react/admin/OrganizationsList.tsx

```diff
@@ -1,344 +1,174 @@
-import type { ChangeEvent, FunctionComponent } from 'react'
-import React, { Fragment, useState } from 'react'
-import { Checkbox, Table, Tag } from 'vtex.styleguide'
-import { useIntl } from 'react-intl'
-import { useRuntime } from 'vtex.render-runtime'
+import React, { useEffect, useState } from 'react'
```

Comment:
Now this is using the new components and I also took the chance to make the component smaller, moving some of the responsibility to their own component file (such as the search logic, the filter logic and the columns creation logic)